daemon/openrazer_daemon/hardware/keyboards.py

```diff
@@ -382,11 +382,12 @@ class RazerBlackWidowV3ProWired(_RippleKeyboard):
                'set_charge_effect', 'set_charge_colour', 'get_battery', 'is_charging', 'set_idle_time', 'set_low_battery_threshold']
 
     DEVICE_IMAGE = "https://dl.razerzone.com/src/3809-1-EN-v1.png"
+    DEVICE_NAME = "Razer BlackWidow V3 Pro (Wired)"
```

Comment:
Please just follow the style from other devices and don't invent a way of doing things